### PR TITLE
Improve console humanized output

### LINKED ISSUE #427: console should provide a command to suggest current space/user
suggest:
yield user()
yield space()
yield server ()

or :
nba >

### PR DESCRIPTION
**Pull request by steppenwolfyuetong**
Mainly improved the following three humanized output:
1)   prompt enhancement, including user, host, current space name
2)  After successfully executing nGQL and outputting the content,  add a blank line
3)  NGQL output, head and data changed to left alignment

close #427

### CODE REVIEW

**Comment by steppenwolfyuetong** [comment]
jenkins go!

**Comment by whitewum** [comment]
would you paste some output images for better explaination?

**Comment by steppenwolfyuetong** [comment]
image

Serial number corresponding to the above.

File: src/console/CliManager.cpp [hunk 1]
```diff
@@ -28,6 +28,7 @@ const int32_t kMaxCommandLineLen = 1024;
 CliManager::CliManager() {
     ::using_history();
     initAutoCompletion();
+    curSpaceName_ = std::move(std::string("(none)"));
```
Comment (dutor) [inline src/console/CliManager.cpp]: no need for `move` here.

File: src/console/CmdProcessor.h [hunk 2]
```diff
@@ -25,9 +25,13 @@ class CmdProcessor final {
     // the method returns true
     bool process(folly::StringPiece cmd);
 
+    std::string getSpaceName();
```
Comment (dutor) [inline src/console/CmdProcessor.h]: `const std::string& getSpaceName() const;`

**Comment by whitewum** [comment]
ref #427

File: src/console/CmdProcessor.h (hunk 2 above)
Comment (steppenwolfyuetong) [reply src/console/CmdProcessor.h]: 👍

File: src/console/CliManager.cpp (hunk 1 above)
Comment (steppenwolfyuetong) [reply src/console/CliManager.cpp]: 👍

**Comment by steppenwolfyuetong** [comment]
jenkins go!

**Comment by steppenwolfyuetong** [comment]
@dutor,@laura-ding  
Thank you for taking care of it. 👍👍

**Comment by whitewum** [comment]
Great Job!

**Pull request merged**